migration/sql-files/093-codebase-add-cve-scan.sql

```diff
@@ -0,0 +1,6 @@
+-- Add new column to codebases cve_scan which keeps track
+-- whether this particular codebase should be scanned or not
+ALTER TABLE codebases ADD COLUMN cve_scan BOOLEAN;
+UPDATE codebases SET cve_scan = 'f';
```

Comment:
Could also use `UPDATE codebases SET cve_scan = false` ;)

Reply:
it is stored as 'f' in the database so used it, do you want me to change it?

Reply:
No, that's fine, both works :)